meinberlin/apps/budgeting/assets/SupportBox.jsx

```diff
@@ -75,13 +83,23 @@ export const SupportBox = (props) => {
     }
   }
 
+  const getSupportClass = () => {
+    if (props.isArchived && props.isReadOnly) {
```

Comment:
I think you can remove the `&& props.isReadOnly`, right? Because when it is archived, it will always be readonly